src/app/game/game-intermediate-result/game-example-drawings/game-example-drawings.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="drawing-container">
+  <app-speech-bubble
+    class="speech-bubble right"
+    [pointerSide]="PointerSide.Top"
+    [arrowAlignment]="ArrowAlignment.Center"
+    [isFlipped]="false"
+    [textColor]="CustomColorsIO.black"
+    [bubbleColor]="CustomColorsIO.teal"
+    [bubbleSize]="'large'"
+  >
+    <div slot="body-alt" class="speech-bubble-text">
+      {{ getMessage() }}

```

Comment:
This should maybe use translation pipe as mentioned in the ts file?
E.g.: `<p class="round-score-text">+ {{ roundScore }} {{ 'POINTS' | translate }}</p>`